cpp/src/bindings.cpp

```diff
@@ -204,6 +204,9 @@ PYBIND11_MODULE(_pypowsybl, m) {
             .value("TWTS", validation_type::TWTS)
             .value("TWTS3W", validation_type::TWTS3W);
 
+    py::enum_<network_modification_type>(m, "NetworkModificationType")
+            .value("VOLTAGE_LEVEL_TOPOLOGY_CREATION", network_modification_type::VOLTAGE_LEVEL_TOPOLOGY_CREATION);
```

Comment:
your enum will have all the other hypothesis ?

Reply:
Yes!